client/static/js/layout.js

```diff
@@ -0,0 +1,69 @@
+const {getAll, getItem, post} = require('./requests');
+
+window.addEventListener('hashchange', updateContent);
+window.addEventListener('load', updateContent);
+
+const form = document.querySelector("#form");
+const postCont = document.querySelector("#post");
+
+document.querySelector("#input-form").addEventListener('submit', post)
+
+document.querySelector("#title").addEventListener('input', labelUpdate)
+document.querySelector("#pseudonym").addEventListener('input', labelUpdate)
+
+// Resets the hash to redirect to the home page
+document.querySelector(".goBack").addEventListener('click', () => {
+    window.location.hash = ''
```

Comment:
when you go back using the go back button, it doesn't clear the values already in the form fields. But that is a simple fix

Reply:
ah i thought i pushed a change to fix that . mustve not been pushed properly